packages/web/localizations/fr.json

```diff
@@ -319,6 +319,7 @@
     "titleLanguage": "Langue",
     "titleShowDust": "Afficher les pools/actifs < 0.01{fiatSymbol}",
     "languages": {
```

Comment:
Please update your branch from `feat/redesign`. I just removed "languages" key from all the json files since I display the name of each language in that language in the menu.

Reply:
Yeah I noticed that right after making a PR, already fixed, the other one too